src/osyris/plot/wrappers.py

```diff
@@ -199,8 +200,11 @@ def line_integral_convolution(ax,
     # Compute line integral convolution
     if length is None:
         length = int(max(z.shape[:-1]) * 15 / 128)
-    with redirect_stderr(io.StringIO()) as _:
```

Comment:
How about using a dummy context manager if `verbose=True`?
E.g.
```Py
from contextlib import redirect_stderr, nullcontext
...
cm = nullcontext() if verbose else redirect_stderr(io.StringIO())
with cm:
    lic_res = lic(z[..., 1], z[..., 0], length=length)
```

Reply:
I wasn't aware we can do that. Thanks

Reply:
`False` by default.